packit_service/worker/parser.py

```diff
@@ -1544,3 +1544,15 @@ def parse_vm_image_build_result_event(event) -> Optional[VMImageBuildResultEvent
             message,
             created_at,
         )
+
+    MAPPING = {
```

Comment:
I wonder if we could incorporate this with the decorators.

Reply:
I'm not sure if I can persuade you, cause I thought about that issue when writing it down :smile: